### PR TITLE
Feature/sanity check for resized finetuning

### PR DESCRIPTION
**Pull request by JeroenMandersloot**
* Refactored experiment pipeline and `params.py`
  - You're now more flexible in specifying how your datasets are split up
* Added augmentation for downscaling the resolution to `finetuning.py`
* Fixed the finetuning code to use a generator for the data
* Added a development version of the `LfwDataset` called `LfwDevDataset`. It consists of a few predefined pairs for which you need to place the two attached files in the `resources/lfw/` directory. It takes a single boolean argument `training` that allows you to toggle between a train and test set.

[pairs_test.txt](https://github.com/HolmesNL/forensische-gezichtsvergelijking/files/4573361/pairs_test.txt)
[pairs_train.txt](https://github.com/HolmesNL/forensische-gezichtsvergelijking/files/4573363/pairs_train.txt)

### CODE REVIEW

File: lr_face/experiments.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
+from datetime import datetime
```
Comment (SimoneAriens) [inline lr_face/experiments.py:1]: Yes, nice! :D

File: lr_face/experiments.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        """
+        if not scorer_names:
+            scorer_names = SCORERS['current_set_up']
+
+        def init_scorer(architecture: Architecture,
+                        tag: Optional[Union[str, Tag]]) -> ScorerModel:
+            if isinstance(tag, str):
+                tag = Tag(tag)
+            # If no version is specified, use latest version.
+            if not tag.version:
```
[the remark this answers is not in the document]
Comment (JeroenMandersloot) [reply lr_face/experiments.py]: Fixed in latest version

File: lr_face/data.py [hunk 3]
```diff
@@ -288,7 +300,8 @@ def images(self) -> List[FaceImage]:
     @cache
     def pairs(self) -> List[FacePair]:
         pairs = []
-        with open(os.path.join(self.RESOURCE_FOLDER, 'pairs.txt'), 'r') as f:
+        pairs_path = os.path.join(self.RESOURCE_FOLDER, self.pairs_file)
+        with open(pairs_path, 'r') as f:
             # The first line tells us how many splits in the data there are,
```
[the remark this answers is not in the document]
Comment (JeroenMandersloot) [reply lr_face/data.py:307]: Zie beschrijving PR, ik heb de twee aangepaste txt files toegevoegd als bijlage. Uiteindelijk moeten we deze even kopieren naar de dataset zodat ze standaard mee komen als iemand de LFW data downloadt.

File: lr_face/data.py [hunk 4]
```diff
@@ -615,7 +664,11 @@ def to_array(data: Union[Dataset,
     # When `data` is a `Dataset` or a list of `FaceImage` instances.
     if isinstance(data, Dataset) or all(
             isinstance(x, FaceImage) for x in data):
-        image_data = [x.get_image(resolution, normalize) for x in data]
+        image_data = [x.get_image(
+            resolution,
+            normalize,
+            augmenter
```
[the remark this answers is not in the document]
Comment (JeroenMandersloot) [reply lr_face/data.py:687]: Heel goed punt! Ik heb dit inmiddels gefixed.
